### DISCUSSION TITLE
Not an issue, just wanted to say thanks

### DISCUSSION THREAD

**Discussion by jackfruh** [6pac/SlickGrid#857] [Show and tell]
Just wanted to say **thanks** to 6pac and others for the continued work on SlickGrid.

I'd have contacted 6pac directly, but there's no contact info for him.

If there's a better way to leave feedback on Github than this, please let me know.

- Jack

**Comment by 6pac** [comment]
Thanks, Jack!
I updated my profile to show an email address, perhaps I'm ready to be slightly less paranoid...

It's great to be able to keep the Slickgrid bandwagon rolling forwards. I do this in my 'spare time' (ha!) and it can feel like i'm dragging the chain a lot of the time. But I do what I can, and welcome anyone else who is happy to step up.

It's hard to gauge where SlickGrid is headed in the bigger picture - so much is changing with javascript frameworks, mobile/touch devices, etc, but I still think it's one of the most flexible and powerful offerings out there. 

It's also hard to gauge how big the user base is. There are a lot of references around the net, but also a lot of offerings in this space. So I'm not sure if this project is a backwater or just waiting to stand out based on the things that matter.

Anyway, it's an integral part of one of my projects, and there are a few surprise extensions lying around on my private fork. One day I'll get to releasing them too. In the meantime, thanks for your words of support.

Ben McIntyre

**Comment by RapidTransit** [comment]
I just came across this, I'm being tortured by other grids, I remember using this one in the past over DataTables. Because it just worked but it was abandoned and the stupid fork graph won't display :( . I'm gonna have to try it out now.

**Comment by ra00l** [comment]
Just wanted to say thanks as well. @6pac would be nice to have a "todo list" with things to improve so we can give a quick hand (I am looking at jquery.* dependencies). 

I have a fork of slick grid with some fixes from the past. Will try, in the next weeks / months, to separate and create some PRs.

**Comment by 6pac** [comment]
Thanks @ra00I that is a great idea. I'll try to put together a 'mind map' of where it's at.
There are different people working on all kinds of things in various clusters. Even I can't keep track of it.
I'm on holidays soon, so might have a bit of time to tinker. I've had about 6 projects on the go at once lately, not a spare second.

**Comment by ghiscoding** [comment]
@6pac 
I also wanted to say thanks for maintaining this repo and also congratulate you on the 1000 :star: milestone that you just passed today 🥇 💁‍♂️

**Comment by ghiscoding** [comment]
Now that we have GitHub Discussions, I think this is better as a Discussion instead of an issue. I would also like to point out that if you haven't noticed, we released SlickGrid 5.0 not long ago and this complete our journey into modernizing the project by dropping jQuery/jQueryUI, migrating to TypeScript, provide ES6/ESM builds and finally we also provided SlickGrid with a fresh new look by adding a new Alpine Theme that can even be customized by CSS/SASS variables.

Thanks for the kind word and keep using SlickGrid 🚀 

--- 
We are also running a quick little poll for fun, thanks for taking the time to participate 

**[What do you think was the most exciting change(s) for you?](https://github.com/6pac/SlickGrid/discussions/853)**

**Comment by ryanhamilton** [comment]
Anyone wishing to say thanks, may I make a suggestion. 
Post a review on your website, blog, social media saying why you like slick grid.
I added one a few years ago and I can see it's steadily forwarded a few people to the project.
